test_aodndata/soop/test_soop_ba.py

```diff
@@ -37,6 +39,7 @@ def setUp(self):
         self.handler_class = SoopBaHandler
         super(TestSoopBaHandler, self).setUp()
 
+    @patch("aodndata.soop.soop_ba.SHIP_CALLSIGN_LS", SHIP_CALLSIGN_LS)
```

Comment:
Can't you add the patch once and for all at the beginning of the class definition?